package.json

```diff
@@ -14,9 +14,12 @@
   "scripts": {
     "postinstall": "ngc -p angular-tsconfig.json",
     "build": "gulp build-release-packages",
+    "bazel:buildifier": "find . -type f \\( -name \"*.bzl\" -or -name WORKSPACE -or -name BUILD -or -name BUILD.bazel \\) ! -path \"*/node_modules/*\" | xargs buildifier -v --warnings=attr-cfg,attr-license,attr-non-empty,attr-output-default,attr-single-file,constant-glob,ctx-args,depset-iteration,depset-union,dict-concatenation,duplicated-name,filetype,git-repository,http-archive,integer-division,load,load-on-top,native-build,native-package,output-group,package-name,package-on-top,redefined-variable,repository-name,same-origin-load,string-iteration,unused-variable",
+    "bazel:format": "yarn -s bazel:buildifier --lint=fix --mode=fix",
```

Comment:
should we change `format` to `ts:format` and make a new `format` that does both?

Reply:
Wanted to do that but was afraid that most people don't need it as the Bazel buildifier goes through all build files (which is kind of slow) and linting is already slow. This is similar to what `angular/angular` does.